Tasks/CondaEnvironmentV1/Tests/L0_conda.ts

```diff
@@ -124,7 +124,7 @@ it('updates Conda if the user requests it', async function () {
     assert(updateConda.calledOnceWithExactly('path-to-conda', Platform.Linux));
 });
 
-it('fails if `conda` is not found', async function (done: MochaDone) {
+it('fails if `conda` is not found', async function () {
```

Comment:
`async` isn't needed here.  I'm going to make a later change to clean up some stuff like this.